### PR TITLE
Add Kawa, an input switcher for OS X.

### PR DESCRIPTION
**Pull request by noraesae**
Hi, I'd like to add a helpful tool to switch inputs with custom shortcuts. Please refer to the following repo for the detail.

https://github.com/noraesae/kawa

Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,6 +106,7 @@
 - [Monodraw](http://monodraw.helftone.com/) - A powerful ASCII art editor.
 - [Subtitles](http://subtitlesapp.com/) - Automatically downloads subtitles for your movies and TV shows.
 - [TheUnarchiver](http://wakaba.c3.cx/s/apps/unarchiver.html) - Unarchive many different kinds of archive files. ![Open-Source Software](media/oss.png)
+- [Kawa](https://github.com/noraesae/kawa) - A better input source switcher with shortcuts. ![Open-Source Software](media/oss.png)
```
Comment (iCHAIT) [inline README.md]: You should place this alphabetically.

Comment (iCHAIT) [inline README.md]: Other than that looks good.

**Comment by noraesae** [comment]
Updated PR with applying alphabetical order.

File: README.md [hunk 2]
```diff
@@ -102,6 +102,7 @@
 - [Finicky](https://johnste.github.io/finicky/) - App that allows you to set rules that decide which browser is opened for every link. ![Open-Source Software][OSS Icon]
 - [Helium](http://heliumfloats.com/) - A floating browser window that allows you to watch media while you work. ![Open-Source Software][OSS Icon]
 - [iStat Menus](https://bjango.com/mac/istatmenus/) - An advanced system monitor for your menubar.
+- [Kawa](https://github.com/noraesae/kawa) - A better input source switcher with shortcuts. ![Open-Source Software](media/oss.png)
```
Comment (iCHAIT) [inline README.md]: Change the link for Open-Source software to [OSS icon]

https://github.com/iCHAIT/awesome-osx/pull/28

**Comment by noraesae** [comment]
Updated again!

**Comment by iCHAIT** [comment]
Thanks :+1:

**Pull request merged**